Monika After Story/game/script-topics.rpy

```diff
@@ -16165,3 +16165,57 @@ label monika_giving_criticism:
     m 3dsu "Graciously accept you can't change everyone's mind and stay considerate when assessing someone else's work."
     m 3hub "...That'd be Monika's Critique Tip of the Day, ahaha!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_curse_words",
+            category=["advice", "life"],
+            prompt="Curse words",
+            random=True,
+            pool=False
+        )
+    )
+
+default persistent._mas_pm_swears_occasionally = None
+
+label monika_curse_words:
+    m 3etd "Say, {w=0.2}[player], {w=0.2}do you ever swear?"
+    menu:
+        m "Say, [player], do you ever swear?{fast}"
+        "Yes.":
+            $ persistent._mas_pm_swears_occasionally = True
+            m 1hksdlb "Ahaha, {w=0,2}I mean, {w=0,2}it's totally fine, {w=0.2}a lot of people do, after all."
+            pass
+        "No.":
+            $ persistent._mas_pm_swears_occasionally = False
+            m 1hub "Ahaha, {w=0.2}I'm really glad then, {w=0.2}because I don't, {w=0.2}either."
+            pass
```

Comment:
```suggestion
        m 2rtd "But, you know, maybe swearing isn't actually so bad?"
        m 3euc "I once heard of some studies stating that using stronger language helps easing the pain if you, say, hurt yourself."

        "Sometimes.":
            $ persistent._mas_pm_swear_frequency = 1
            m 1hub "Oh, only sometimes? That's probably just fine then. Most people have probably sworn at least at some point in their life."
        m 3euc "I even heard of some studies stating that using stronger language helps easing the pain if you, say, hurt yourself."

Reply:
Sorry, I totally forgot to remove `pass`es here...

Reply:
You also forgot to commit the "Sometimes.": menu option in, along with its dialogue. Unless that was intentional?